auth-validation.js

```diff
@@ -0,0 +1,142 @@
+// 비밀번호 관련 기능 모듈 가져오기
+import { pw } from "./toggle-pw.js";
+
+// DOM 요소 참조
+const form = document.querySelector(".js-form");
+const input = document.querySelectorAll(".js-input");
+const button = document.querySelector(".js-button");
+const emailError = document.querySelector(".js-email-error");
+const pwError = document.querySelector(".js-pw-error");
+const nicknameError = document.querySelector(".js-nickname-error");
+const pwCheckError = document.querySelector(".js-pw-check-error");
+const pwCheck = document.querySelector(".js-pw-check");
+
+// 유효성 검사 결과
+const PASSED = 0;
+const FAILED = 1;
+let valueStatus = [];
+
+// 이벤트 핸들러
+function valueCheck(e) {
+  !e.target.value ? setInvalid(e.target) : setValid(e.target);
+}
```

Comment:
P2:
함수의 경우 동사가 먼저 오는 것이 일반적이라 아래와 같은 이름을 추천드립니다.

```suggestion
function checkValue(e) {
  !e.target.value ? setInvalid(e.target) : setValid(e.target);
}
```